src/Jack/Combinators.purs

```diff
@@ -75,8 +75,16 @@ scale f j =
 -- | Generates a 'Char' in the given range.
 chooseChar :: Char -> Char -> Gen Char
 chooseChar x0 x1 =
-  map fromCharCode $
-    chooseInt (toCharCode x0) (toCharCode x1)
+  let
+    fromInt x =
+      case fromCharCode x of
```

Comment:
`Data.Char.fromCharCode` changed its api to be `Int -> Maybe Char`. The way we're handling it here is to default to one of the given `Char`s. More than happy to rethink this change if it seems wrong.

Maybe add a `chooseChar` to `Jack.Random`? Maybe throw because it should never be out of bounds? Maybe do something else?